project_0a/project_0a_4.c

```diff
@@ -0,0 +1,35 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 4 - build a C to F converter. F = C × 1.8 + 32. Use scanf("<type>",
+&<variable>) to accept input from keyboard.
+*/
+
+#include <stdio.h>
+
+int main() {
+  // NOTE: Already made exercise 1 with scanf() input
```

Comment:
This note should be at the top of the file, it doesn't have anything to do with the `main` function.

If the same file solves two exercises, just name it as such instead of copying it. It bloats the review, and forces you to apply every fix twice, for no good reason.